pkg/storage/image_test.go

```diff
@@ -52,18 +52,12 @@ var _ = t.Describe("Image", func() {
 		mockCtrl.Finish()
 	})
 
-	mockParseStoreReference := func() {
-		inOrder(
-			mockStorageReferenceStringWithinTransport(storeMock),
-		)
-	}
-
-	mockGetRef := func() {
-		gomock.InOrder(
+	mockGetRef := func() mockSequence {
+		return inOrder(
 			storeMock.EXPECT().Image(gomock.Any()).
 				Return(&cs.Image{ID: testImageName}, nil),
+			mockParseStoreReference(storeMock, ""), // FIXME: What image name is expected?
```

Comment:
I would remove that `FIXME` since I guess the image name does not matter at all for these kind of tests, right?

Reply:
The name does matter now, see how `mockParseStoreReference` uses it.

Note that this `FIXME` does not remain at the end of this PR, the commit “Fix mockGetRef, and deal with all of the fallout” removes it.

(I do generally try to make my PRs close to a sequence of one-step-at-a-time commits with nothing left to fix up later, in this case it was a bit tricky because `mockNewImage` needs to be created and use the new stricter `mockParseStoreReference` for the “Fix mockGetRef, and deal with all of the fallout” commit to be possible.

OTOH, it _is_ possible not to have that FIXME even temporarily, by temporarily introducing two variants of `mockParseStoreReference` in the worst case, or maybe by taking more care to split “Fix mockGetRef, and deal with all of the fallout” into individual precisely-reasoned steps, that commit is rather a collection of whatever remained outstanding. Let me know if I should work on that.